src/Core/ProtocolDefines.h

```diff
@@ -72,3 +72,5 @@
 #define DBMS_MIN_PROTOCOL_VERSION_WITH_SERVER_QUERY_TIME_IN_PROGRESS 54460
 
 #define DBMS_MIN_PROTOCOL_VERSION_WITH_PASSWORD_COMPLEXITY_RULES 54461
+
+#define DBMS_MIN_PROTOCOL_VERSION_WITH_TIMEZONE_UPDATES 54461
```

Comment:
copy / paste error ?
expected 54461 ?

Reply:
Not erro, but placeholder. I dont know yet what will be the final revision - master changes constantly and rapidly :) I think its to be changed right before merge